lib/tags/profile-settings.tag

```diff
@@ -435,14 +445,18 @@
                             self.refs.password.classList.remove('graphjs-error');
                             self.refs.confirmation.classList.remove('graphjs-error');
                             self.refs.password.classList.add('graphjs-success');
+                            self.refs.password.setAttribute('data-savesuccesstext', i18n.saveSuccessText);
```

Comment:
Please remove this change. self.refs.password refers to an input element and input elements do not accept ::before or ::after pseudo-elements.